pkg/eventprocessor/impl/executor/impl/txnscope_runsql.go

```diff
@@ -220,7 +220,17 @@ func (ts *txnScope) executeWriteStmt(
 	addr common.Address,
 	policy tableland.Policy,
 	beforeRowCount int,
+	isOwner bool,
 ) error {
+	if ws.Operation() == tableland.OpAlter {
```

Comment:
only the owner can execute the ALTER TABLE operation. that's the only ACL rule we have right now. ALTER TABLE is not part of the GRANT/REVOKE permissions and neither part of the policy